### ISSUE TITLE
Org mode to Beamer "institute" metadata field escaping latex

### ISSUE THREAD

**Issue by jessejenks** [jgm/pandoc#6387]
When converting to a beamer presentation from Org mode, the "institute" field seems to be escaping latex like `\inst{1}` while other fields do not. However, this doesn't seem to happen in markdown. I am guessing this is not a difficult fix, possibly just [adding a case to the `metaValue` function](https://github.com/jgm/pandoc/blob/master/src/Text/Pandoc/Readers/Org/Meta.hs#L74)?

My current workaround is to use the "latex header" option.

Examples:
- [Org mode](https://pandoc.org/try/?text=%23%2BTITLE%3A+My+Title%0A%23%2BAUTHOR%3A+First+Author+%5Cinst%7B1%7D%2C+Second+Author+%5Cinst%7B2%7D%0A%23%2BINSTITUTE%3A+%5Cinstitute%7B+%5Cinst%7B1%7D+First+Institute+%5C%5C+%5Cinst%7B2%7D+Second+Institute+%7D%0A%0A*+First+Slide%0ASome+text%0A%0A*+Last+Slide%0AThank+you+for+coming+to+my+talk&from=org&to=beamer&standalone=1)

Produces
```latex
\title{My Title}
\author{First Author \inst{1} \and Second Author \inst{2}}
\date{}
\institute{\textbackslash institute\{ \textbackslash inst\{1\} First
Institute \textbackslash\textbackslash{} \textbackslash inst\{2\} Second
Institute \}}
```

- [Markdown]()

Produces
```latex
\title{My Title}
\author{First Author \inst{1}, Second Author \inst{2}}
\date{}
\institute{\inst{1} First Institute, \inst{2} Second Institute}
```

- [Org mode workaround](https://pandoc.org/try/?text=%23%2BTITLE%3A+My+Title%0A%23%2BAUTHOR%3A+First+Author+%5Cinst%7B1%7D%2C+Second+Author+%5Cinst%7B2%7D%0A%23%2BLATEX_HEADER%3A+%5Cinstitute%7B+%5Cinst%7B1%7D+First+Institute+%5C%5C+%5Cinst%7B2%7D+Second+Institute+%7D%0A%0A*+First+Slide%0ASome+text%0A%0A*+Last+Slide%0AThank+you+for+coming+to+my+talk&from=org&to=beamer&standalone=1)

Produces
```latex
\institute{ \inst{1} First Institute \\ \inst{2} Second Institute }

\title{My Title}
\author{First Author \inst{1} \and Second Author \inst{2}}
\date{}
```

Thanks for making such a great piece of software!

- Pandoc version: 2.9.2.1

**Comment by tarleb** [comment]
I'm a bit hesitant to support a field that's not defined by Emacs org-mode, but I think it makes sense in this case. Your code analysis is correct, would you like to submit a PR?

**Comment by tarleb** [comment]
We might also want to add support for [Beamer specific export settings](https://www.gnu.org/software/emacs/manual/html_node/org/Beamer-specific-export-settings.html).

**Comment by tarleb** [comment]
Fixed in d2d5eb8a993553eb5d1b9b9ca7863638864dde7e
